### ISSUE TITLE
NIX Load Limit HTTP Status Code

### ISSUE THREAD

**Issue by RaidForums** [mybb/mybb#4352]
Just a suggestion I guess, but you shouldn't have the NIX Load Limiting setting give users a HTTP 404 code when it's triggered as this is also horrible for SEO.

I suggest the HTTP Code 503, you could add a simple bypass so that spiders/crawlers don't get caught by this setting but it will add a bypass method for malicious users to just change their user agent and bypass the nix load limit setting.

**Comment by Stefan-MyBB** [comment]
While a 503 would make sense I am unable to reproduce this bug report. I don't get a 404 status code.

**Comment by RaidForums** [comment]
> 
> 
> While a 503 would make sense I am unable to reproduce this bug report. I don't get a 404 status code.

So, what http error code do you receive when you get a NIX Load limit page?

**Comment by euantorano** [comment]
As far as I can see from [the current code](https://github.com/mybb/mybb/blob/da7b32456e7b21c96f72c1093816c55cce249264/global.php#L1147), no specific status code is sent at all.

A 503 status code would make sense. The core doesn't do a great job of setting status codes at all. A quick and easy fix would be to add an optional parameter to the `error` function to allow a status code to be defined.

**Comment by Sama34** [comment]
Do you use Google SEO ?

**Comment by RaidForums** [comment]
> 
> 
> Do you use Google SEO ?

Yes it probably has something to do with it, but still this requires some work. Add a status code option to the error function.
